tests/runfiles/linux.run

```diff
@@ -237,35 +237,22 @@ tests = ['zpool_attach_001_neg', 'attach-o_ashift']
 [tests/functional/cli_root/zpool_clear]
 tests = ['zpool_clear_002_neg', 'zpool_clear_003_neg']
 
-# DISABLED:
-# zpool_create_001_pos - needs investigation
-# zpool_create_002_pos - needs investigation
-# zpool_create_004_pos - needs investigation
-# zpool_create_006_pos - https://github.com/zfsonlinux/zfs/issues/3484
-# zpool_create_008_pos - uses VTOC labels (?) and 'overlapping slices'
-# zpool_create_011_neg - tries to access /etc/vfstab etc
-# zpool_create_012_neg - swap devices
-# zpool_create_014_neg - swap devices
-# zpool_create_015_neg - swap devices
-# zpool_create_016_pos - no dumadm command.
-# zpool_create_020_pos - needs investigation
-[tests/functional/cli_root/zpool_create]
```

Comment:
> Warning: Test 'zpool_create_*' removed from TestGroup '/usr/share/zfs/zfs-tests/tests/functional/cli_root/zpool_clear' because it failed verification.

It took me a while to realize the cause: without `[tests/functional/cli_root/zpool_create]` all these tests are now part of the `zpool_clear` group.